### ISSUE TITLE
Missing aria-label attributes on 'Manage Proxies' link

### ISSUE THREAD

**Issue by h-parekh** [samvera/hyrax#3041]
### Descriptive summary
'Manage Proxies' link on [dashboard page](https://nurax.curationexperts.com/dashboard?locale=en) doesn't have aria-labels. 
<img width="533" alt="screen shot 2018-05-07 at 1 27 02 pm" src="https://user-images.githubusercontent.com/21689371/39715181-68a1cb80-51fa-11e8-90f2-2ae9ef7aa405.png">

### Rationale

Add screen reader only text or an aria-label attribute to each link which tells users where it goes and/or what it does. 

### Expected behavior
Screen reader should be able to read to users about the link

### Actual behavior
No aria-labels are present.
Tested in Hyrax Version: 2.1.0.rc1 
Nurax Version
ENV: production 
SHA: 885c4a6c71a8e67c9a1e7ed506be9db2fce51b8b 
Branch: master 

### Related work
Violates [WCAG 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html) guidelines

**Comment by adamjarling** [comment]
Looks like this link might have been removed.  I'm not seeing it on either Nurax or local Hyrax:

![dashboard-manage-proxies-nurax](https://user-images.githubusercontent.com/3020266/45309542-87970080-b4e9-11e8-80bc-3a51a75f8089.png)

![dashboard-manage-proxies](https://user-images.githubusercontent.com/3020266/45309543-87970080-b4e9-11e8-8145-d9ed6ccaba97.png)

**Comment by kdid** [comment]
@adamjarling - this shows on the Dashboard page itself, when the user is not an admin
